test/e2e/vsphere_autoimport_test.go

```diff
@@ -163,4 +163,4 @@ func TestVSphereKubernetes123BottlerocketAutoimport(t *testing.T) {
 		framework.WithEnvVar(features.K8s123SupportEnvVar, "true"),
 	)
 	runAutoImportFlow(test, provider)
-}
\ No newline at end of file
+}
```

Comment:
??

Reply:
gofumpt added this

Reply:
I'm fine if you want to change this, but it should be in a different PR, this one is not even touching that package

Plus, if the linter is not complaining, we shouldn't reformat random files